package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "generator-template-ui5-project",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
not sure whether this will be auto-bumped by the release process. just noting that here, don't think it'll cause hick-ups